### PR TITLE
[CDAP-12455] Fixes the positioning of second and third level popover position

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-12455

Right now the positioning of popover menus is not consistent across browsers. Have filed a JIRA for 4.3.1 as its not a blocker: https://issues.cask.co/browse/CDAP-12468

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/helper.js [hunk 1]
```diff
@@ -45,19 +45,21 @@ export function isCustomOption(selectedOption) {
...
   let elem = element;
   let elemBounding = elem.getBoundingClientRect();
-  const FOOTER_HEIGHT = footerHeight;
 
-  let popover = document.getElementsByClassName('second-level-popover');
+  let popover = document.getElementsByClassName(popoverLevel);
   let popoverHeight = popover[0].getBoundingClientRect().height;
   let tableContainerScroll = document.getElementById('dataprep-table-id').scrollTop;
   let popoverMenuItemTop = elemBounding.top;
-  let bodyBottom = document.body.getBoundingClientRect().bottom - FOOTER_HEIGHT;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/helper.js:57]: I believe there was a problem with the calculation if you remove this footer height if the second level popover is after the scrollable popover item on the main popover. ie. smaller screen, and `Define Variable`

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/DataPrep/helper.js:57]: @elfenheart I tried all the directives, second level and third level and this is what I am seeing. I am not sure what changed for some the `document.body.getBoundingClientRect` returns bottom that doesn't include the footer. So the removing the footer height from the bottom seemed un-necessary. 

This screenshot was taken in a browser size of 1000x600
screen shot 2017-08-24 at 1 46 33 pm


Let me know if I missed any usecase.

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Review by tonybach — APPROVED** [review]
LGTM 👍

**Comment by leacd** [comment]
LGTM

**Pull request merged**